src/cmd/export.rs

```diff
@@ -16,6 +16,11 @@ pub fn get_subcommand<'a, 'b>() -> App<'a, 'b> {
         .settings(&SUBCMD_SETTINGS)
         .arg(Arg::with_name("type")
             .help("Type of export.\n- html: static html. Writes to `./index.html` and `./css/`"))
+        .arg(Arg::with_name("output-directory")
+            .short("d")
```

Comment:
Wouldn't do that, honestly. I'd use `-o` or `--output` for single files, not for directories.